models/repo_watch_test.go

```diff
@@ -59,15 +59,15 @@ func TestRepository_GetWatchers(t *testing.T) {
 	assert.NoError(t, PrepareTestDatabase())
 
 	repo := AssertExistsAndLoadBean(t, &Repository{ID: 1}).(*Repository)
-	watchers, err := repo.GetWatchers(1)
+	watchers, err := repo.GetWatchers(ListOptions{Page: 1, PageSize: ItemsPerPage})
```

Comment:
why do you have to change PageSize ...

https://github.com/go-gitea/gitea/pull/9452/files#diff-3f54f4b6a38724be14a39245ceb172a0L161-L162